### PR TITLE
EN-10522: removed old API handlers

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
removed old API handlers and small fixes in tests

### CODE REVIEW

File: api/middleware/endpointThrottler.go [hunk 1]
```diff
@@ -14,61 +14,6 @@ type throttlerGetter interface {
 	GetThrottlerForEndpoint(endpoint string) (core.Throttler, bool)
 }
 
-// CreateEndpointThrottler will create a middleware-type of handler to be used in conjunction with special
-// REST API end points that need to be better protected
-func CreateEndpointThrottler(throttlerName string) gin.HandlerFunc {
```
Comment (iulianpascalau) [inline api/middleware/endpointThrottler.go:19]: Do we have something instead of this?

Comment (bogdan-rosianu) [reply api/middleware/endpointThrottler.go:19]: yes, in previous PRs I have added `CreateEndpointThrottlerFromFacade` that remained. There are also unit tests that prove the functionality is still working

Comment (iulianpascalau) [reply api/middleware/endpointThrottler.go:19]: great

**Review by iulianpascalau — APPROVED** [review]

File: api/middleware/middleware.go [hunk 2]
```diff
@@ -1,17 +1,5 @@
 package middleware
 
-import (
-	"github.com/gin-gonic/gin"
-)
-
 // Handler interface defines methods that can be used from `facade` context variable
 type Handler interface {
```
Comment (BeniaminDrasovean) [inline api/middleware/middleware.go:4]: This does not seem to be used anywhere. Can it be removed?

**Review by BeniaminDrasovean — DISMISSED** [review]

**Review by BeniaminDrasovean — DISMISSED** [review]

**Review by iulianpascalau — DISMISSED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.
image

**Review by iulianpascalau — DISMISSED** [review]

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**